config/servicewatchdog/services_servicewatchdog_add.php

```diff
@@ -55,7 +55,7 @@
 
 if ($_POST) {
 	if (!is_numeric($_POST['svcid']))
```

Comment:
That one is odd. Given what it's testing that should probably have a "return;" on the line after it so that a non-numeric service ID is skipped.